ballista/rust/executor/src/main.rs

```diff
@@ -109,8 +108,10 @@ async fn main() -> Result<()> {
         info!("Running in local mode. Scheduler will be run in-proc");
         let client = StandaloneClient::try_new_temporary()
             .context("Could not create standalone config backend")?;
-        let server =
-            SchedulerGrpcServer::new(SchedulerServer::new(Arc::new(client), namespace));
+        let server = SchedulerGrpcServer::new(SchedulerServer::new(
+            Arc::new(client),
+            "ballista".to_string(),
```

Comment:
Why do we still need to provide the namespace here?

Reply:
I checked the code out locally and see that this is where we are running in local mode. Makes sense.